src/Collection/Collection.router.ts

```diff
@@ -282,6 +291,7 @@ export class CollectionRouter extends Router {
       return {
         collection: toFullCollection(result.collection),
         items: result.items,
+        itemCurations: result.itemCurations,
```

Comment:
Adding this here because we need to keep track of the `itemCurations` in the FE as well, so now it's part of the response.